lib/puppet/functions/foreman/ensure_value_in_string.rb

```diff
@@ -0,0 +1,28 @@
+# Returns an string with appended values to the end if they were not present in original string.
```

Comment:
Now that we have structured facts I'm wondering if we should make https://github.com/theforeman/puppet-foreman/blob/3abb8b1ddb083c289e682a040f6e46e745f2cf4a/lib/facter/sssd.rb return one ```sssd``` fact (and one ```ipa``` fact) with all the properties. That would allow us to return a proper array and do this in pure puppet code without a function.

Doesn't have to be part of this PR but if we're cleaning up we might as well do it properly.

Reply:
I would prefer it if we could stick to migrating the functions in this PR as I'm not too familiar with these parts.

Reply:
https://github.com/theforeman/puppet-foreman/pull/618 should remove the need for this function altogether.